src/OrchardCore.Modules/OrchardCore.Contents/Views/Admin/List.cshtml

```diff
@@ -52,7 +53,7 @@
                     <div class="dropdown-menu dropdown-menu-right" aria-labelledby="new-dropdown">
                         @foreach (var item in Model.Options.CreatableTypes)
                         {
-                            <a class="dropdown-item" href="ContentTypes/@item.Value/Create">@item.Text</a>
+                            <a class="dropdown-item" href="@formActionCreate/@item.Value/Create">@item.Text</a>
                         }
```

Comment:
@agriffard too late, but can we do this with the anchor tag helper?
Otherwise using `PathBase` like this can cause encoding problems.
```
<a asp-action="Create" asp-route-id=@item.Value>@item.Text</a>
```
That should handle the pathbase automatically and build it from the urlhelper

Reply:
good catch, i will fix it in #4479 after merging dev.

Reply:
In fact, the problem is when you get an encoded url (if `PathString.ToString()` calling `ToUriComponent()` is implictly / explicitly called) where you think it is not, e.g when you check if all or part of a path matches some unencoded strings.

Yes, here `PathString.ToString()` is implictly called

    var formActionCreate = Context.Request.PathBase.Value + "/Admin/Contents/ContentTypes";

But it is just to provide an `href` and, because we are doing the same for `formActionWithoutTypeName` but in a script, that's OK to not use the urlHelper (e.g through the anchor tag helper), and let the browser encode the url.

But, as a general rule, i think it is better to encode the whole path or not at all, but not only a part of it.

Encoding the whole would be done by 1st initializing the var as a `PathString` (not a string) in one of these 2 ways.

    var formActionCreate = Context.Request.PathBase+ new PathString("/Admin/Contents/ContentTypes");
    var formActionCreate = Context.Request.PathBase.Add("/Admin/Contents/ContentTypes");

And then use it like this

    href="@formActionCreate.Add("/" + @item.Value).Add("/Create")"

But, because we are doing the same for `formActionWithoutTypeName` but in a script, in my PR #4479 i opted to not encode the path at all by using `PathBase.Value` when initializing both variables.

    var formActionWithoutTypeName = Context.Request.PathBase.Value + "/Admin/Contents/ContentItems";
    var formActionCreate = Context.Request.PathBase.Value + "/Admin/Contents/ContentTypes";